### DISCUSSION TITLE
Tips for Improving DeepAR model performance !

### DISCUSSION THREAD

**Discussion by sayedathar11** [awslabs/gluonts#1806] [Q&A] [answered]
Hiii,
We are building a time series model that predicts target in range 0 to 1 , We have close to 37k time series for training , All of this time series need not be of same length necessarily and are feed to DeepAR model , currently our model is stuck on MAE score of 0.10 , we would like to know some tips , tricks or suggestions that could come in handy to improve performance of this Deep AR model .
@mbohlkeschneider @lostella  any suggestions , Please do help .
Thanks 
Athar

**Comment by mbohlkeschneider** [comment, marked as answer]
Hi @sayedathar11,

Thank you for your question. It is extremely difficult to give specific advice without access to the dataset. It is also hard to tell whether an MAE is good or bad (depending on how difficult the dataset is). 

I would suggest the following steps: 

1. Run a very simple model (moving average or naive seasonal) to get a baseline MAE on your dataset. 

2. From this, you should expect a 5-10% improvement for a model like DeepAR. In very favorable cases, this might go up to 30%. This gives you an estimate on how much improvement you can expect. 

3. Do a error analysis. Look at the forecasts that are really bad and form a hypothesis to improve the model. 

This is what I would do. Again, specific advice is difficult to give without seeing the data.
